app/src/main/java/org/jellyfin/androidtv/ui/global/NetworkStatusViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package org.jellyfin.androidtv.ui.global
+
+import androidx.lifecycle.ViewModel
+import org.jellyfin.androidtv.data.repository.NetworkState
+import org.jellyfin.androidtv.data.repository.NetworkStatusRepository
+
+class NetworkStatusViewModel (
+	private val repo: NetworkStatusRepository
+) : ViewModel() {
+	val getNetworkState = repo.state
+	fun getCurrentNetworkState() : NetworkState = repo.state.value
```

Comment:
We probably want to use LiveData here so we can dynamically show the connection state in the toolbar/other places at some point.

Reply:
So I went for StateFlow as the official `AndroidDevelopers` blog are nudging people towards `StateFlows`: https://medium.com/androiddevelopers/migrating-from-livedata-to-kotlins-flow-379292f419fb

I'm tempted to drop `getCurrentNetworkState` (which is unused) and we can simply have two methods which returns a `StateFlow` or `LiveData` depending on requirements? 
The former seems to be preferred as the Fragment can write lifecycle aware handling (using `repeatOnLifestyle`) by the look of it.

Reply:
I was not aware that LiveData is not recommended anymore. I already preferred using flows so that's a win-win :p.

I think dropping the function is fine, just expose the flow from the viewmodel so we can use it from the activity/fragment/other lifecycle. No need for livedata.